src/App/FreeCADInit.py

```diff
@@ -33,10 +33,25 @@
 # imports the one and only
 import FreeCAD
 
+def removeFromPath(module_name):

```

Comment:
I might be misunderstanding things, but it seems like this function isn't really used anywhere (except on line 49 but without an actual `module_name` as argument)? So I guess this can be removed?